app/modules/usage_stats/migrations/2018_02_08_000001_usage_stats20180208000001.php

```diff
@@ -0,0 +1,260 @@
+<?php
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class UsageStats20180208000001 extends Migration
+{
+    
+    private $tableName = 'usage_stats';
+    private $tableNamebackup = 'usage_stats_backup';
+    
+    public function up()
+    {
+        $capsule = new Capsule();
+        $migrateData = false;
+        
+        if ($capsule::schema()->hasTable($this->tableName)) {
```

Comment:
As this is the second migration, this will always be true, so the if statement can safely be removed (unless you want to cater for people deleting the table)